app/assets/javascripts/ci_api/controllers/CommonChatController.js

```diff
@@ -124,6 +140,14 @@ export default class CommonChatController {
         return document.getElementById("HMRC_CIAPI_Embedded_1")
     }
 
+    _getPopupDiv() {
+        return document.getElementById("HMRC_CIAPI_Fixed_1")
+    }
+
+    _getAnchoredPopupDiv() {
```

Comment:
How is HMRC_CIAPI_Popup_1 div tag covered?

Reply:
As discussed with Neha, this scenario has not been developed yet. So  not needed for now